docs/timelines/running-and-editing-timelines.md

```diff
@@ -0,0 +1,135 @@
+
+1.  Ensure the **timecode link** button (bottom left of the timeline view) is set to **Linked**.
+
+2.  Push up the fader of the handle where the timeline is stored.
+
+3.  Start the timecode source. 
+
+4.  The cues in the timeline will run when the timecode reaches their programmed times.
+
+- On consoles with motorised faders and on virtual faders, the faders will move as programmed.
+
+- If you are using internal timecode you can set the timecode to run automatically when you raise the fader, and/or to stop when you lower the fader. See [Timeline Options - Fader](../timelines/timeline-options.md#fader-tab).
+
+- You can set the timeline to loop (when using internal timecode), see [Timeline Options - Timecode](../timelines/timeline-options.md#timecode-tab).
+
+- If you start the timecode at a point part way through the timeline, any active triggers will fire, but previous triggers will not - this might result in fixtures starting from a different state and changing the look.
+
+### Testing a timeline
+
+You can test a timeline without running the timecode source by setting the **timecode link** button to **Unlinked**. You can then use the Play/Pause/Rewind buttons
+in the Timeline View window to run the timeline sequence. This can be useful when you don't have control of the timecode source.                  
+
+Timeline Editing
+--------------------------
+ 
+### Selecting triggers
+
+To edit a trigger, you need to select it. There are several different ways to do this.
+-   Click on a trigger block to select all the trigger points within the block. You will see selection 
+    handles appear.  Click again on a selected trigger block to step through the
+    trigger points within the block. 
+
+![Timeline trigger handles](/docs/images/Timeline-Handles.png)
+
+- Click on more triggers to add them to the selection.
+
+- Draw a selection marquee box over the trigger points you want to select. Only the trigger points enclosed by the selection box will be selected, so if you want to select all points in a block they all need to be inside the box. This is a quick way to select one trigger point without having to "multi-click" on a block.
+
+- To clear all selection use the context menu button \{Clear Trigger Selection\} or click in an empty area of the Timeline View.
+
+### Using arrow buttons to move trigger selection
+
+The left/right arrow buttons for the track will move your selection to the previous trigger point or the next trigger point.
+
+- Left arrow - moves selection to the previous trigger point on the track.
+- Right arrow - moves selection to the next trigger point on the track.
+
+If nothing is selected, the right arrow will select the first trigger on the track and the left arrow will select the last.
+
+### Using the wheels to edit triggers
+
+When you have one or more trigger points selected, the wheels control the time, level and fade of those triggers.
+
+- Left hand wheel sets the time.
```

Comment:
This will vary depending on what type of trigger you have selected. I think you're right that this is the only option now though.